sc/core/keywords.py

```diff
@@ -0,0 +1,17 @@
+
+class Labels:
+
+    SC_NODE = "sc_node"
+    SC_LINK = "sc_link"
+    SC_EDGE = "sc_edge"
+    SC_ARC = "sc_arc"
+    SC_EDGE_SOCK = "sc_edge_socket"
+
+
+class TypeAttrs:
```

Comment:
Why are these types selected? What do they mean?

Reply:
Just read code please. This is name of attributes:

`TypeAttrs.CONST` - this is a name of property. Bu possible values: `CONST`, `VAR`, `UNKNOWN`

Reply:
https://github.com/deniskoronchik/neo4j-platform/blob/extend/sc/core/types.py